### PR TITLE
Write code for TC10

### PR DESCRIPTION
**Pull request by lqthai07t3**
- Code for TC10.
- Run code and check.

### CODE REVIEW

File: src/main/java/pages/ForgotPasswordPage.java [hunk 1]
```diff
@@ -9,11 +9,13 @@
     private By txtEmail = By.id("email");
     private By lnkForgotPassword = By.cssSelector("a[href*='ForgotPassword']");
     private By btnSendInstructions = By.cssSelector("input[value='Send Instructions']");
+    private By lblErrorMessage = By.className("error");
```
Comment (anhlvt) [inline src/main/java/pages/ForgotPasswordPage.java:12]: Should we catch locator more clearly?

Comment (lqthai07t3) [reply src/main/java/pages/ForgotPasswordPage.java]: Updated locator. Please check again :)

File: src/test/java/tests/ForgotPasswordTests.java [hunk 2]
```diff
@@ -24,4 +24,19 @@ public void tc09_ResetPasswordWithValidEmail() {
...
         Assert.fail("Browser direct to error page with message: Mailbox unavailable. The server response was: that smtp username is not allowed to send");
     }
+
+    @Test(testName = "User can't reset password if enter incorrect email address")
+    public void tc10_ResetPasswordWithInvalidEmail() {
+        LogHelper.info("Go to Login Tab");
+        homePage.clickLoginTab();
+
+        LogHelper.info("Click on Forgot Password link");
+        forgotPasswordPage.clickForgotPasswordLink();
```
Comment (MinhQuan119) [inline src/test/java/tests/ForgotPasswordTests.java:34]: image
clickForgotPasswordLinkmethod belongs to LoginPage class, not ForgotPasswordPage

Comment (lqthai07t3) [reply src/test/java/tests/ForgotPasswordTests.java]: Yeah, I has fixed it. Thank you so much

**Review by MinhQuan119 — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

**Pull request merged**